doc/dev_guide/create_aws_role.sh

```diff
@@ -15,8 +15,9 @@ ROLE_NAME="shira-28-11" # role name that you pick in your AWS account (replace s
 NAMESPACE="test1" # namespace name where noobaa will be running (replace test1 with your value)
 
 # noobaa variables
-SERVICE_ACCOUNT_NAME_1="noobaa" # The service account name of statefulset core and deployment operator
+SERVICE_ACCOUNT_NAME_1="noobaa" # The service account name of deployment operator
 SERVICE_ACCOUNT_NAME_2="noobaa-endpoint" # The service account name of deployment endpoint
+SERVICE_ACCOUNT_NAME_3="noobaa-core" # The service account name of statefulset core
```

Comment:
I agree with you, at this point I prefer the least needed changes for it to work since the docs are already up (and in the original instructions that we had from the team of the CCO there was one variable).
I will not change at this point.